examples/io_perf_test.cpp

```diff
@@ -0,0 +1,176 @@
+#include <iostream>
+
+#include "opentimelineio/clip.h"
+#include "opentimelineio/typeRegistry.h"
+#include "util.h"
+#include <opentimelineio/any.h>
+#include <opentimelineio/serialization.h>
+#include <opentimelineio/deserialization.h>
+#include <opentimelineio/timeline.h>
+
+namespace otio = opentimelineio::OPENTIMELINEIO_VERSION;
+
+using chrono_time_point = std::chrono::steady_clock::time_point;
+
+const struct {
+    bool PRINT_CPP_VERSION_FAMILY    = false;
+    bool TO_JSON_STRING              = true;
+    bool TO_JSON_STRING_NO_DOWNGRADE = true;
+    bool TO_JSON_FILE                = true;
+    bool TO_JSON_FILE_NO_DOWNGRADE   = true;
+    bool CLONE_TEST                  = true;
+    bool SINGLE_CLIP_DOWNGRADE_TEST  = true;
+} RUN_STRUCT ;
+
+/// utility function for printing std::chrono elapsed time
+const void
```

Comment:
Minor point, the "const" is probably not necessary.

Reply:
I think I've addressed these notes.  Thanks @darbyjohnston -- let me know if you see anything else or if I missed anything!